CHANGELOG.md

```diff
@@ -1,4 +1,29 @@
 ## Unreleased
+
+## [2.6.0] - 2017-01-16
+### Added
+- cli: log processor pool total on shutdown
+- amqp_job: meter for job finish state (#184)
+- amqp_job: track queue time via queued_at field from payload
+- amqp_job_queue: log job id immediately after json decode
+- capture every requeue and send the error to sentry
+
+### Changed
```

Comment:
Should we include that the SSH code was consolidated to be shared by all the backends? In theory it should be a no-op, but there were minor differences in how each of the backends SSH-ed, so there might be some minor behavior differences.